easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.1.1-34-GCCcore-13.2.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.1.1-34'
+
+homepage = 'https://www.imagemagick.org/'
+description = "ImageMagick is a software suite to create, edit, compose, or convert bitmap images"
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/%(name)s/%(name)s/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['19f4303774b56be182c576b266c34bc824fcaef1d1d243192344d015adb0ec28']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '2.0.3'),
+]
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('X11', '20231019'),
+    ('Ghostscript', '10.02.1'),
+    ('JasPer', '4.0.0'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('LibTIFF', '4.6.0'),
+    ('LittleCMS', '2.15'),
+    ('Pango', '1.51.0'),
+    ('pixman', '0.42.2'),
+    ('FriBidi', '1.0.13'),
+]
+
+configopts = "--with-gslib --with-x"
```

This seems to work for me:
```suggestion
preconfigopts = 'PKG_CONFIG=$EBROOTPKGCONF/bin/pkgconf'
configopts = "--with-gslib --with-x"
```